docs/cloud-storage-fileset-example.md

```diff
@@ -0,0 +1,678 @@
+
+The following table demonstrates which jars are necessary for different cloud storage filesets:
+
+| Hadoop runtime version | S3                                                                                                                                       | GCS                                                                                                          | OSS                                                                                                                                        | ABS                                                                                                                |
+|------------------------|------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| No Hadoop environment  | `gravitino-aws-bundle-${gravitino-version}.jar`                                                                                          | `gravitino-gcp-bundle-${gravitino-version}.jar`                                                              | `gravitino-aliyun-bundle-${gravitino-version}.jar`                                                                                         | `gravitino-azure-bundle-${gravitino-version}.jar`                                                                  |
+| 2.x, 3.x               | `gravitino-aws-${gravitino-version}.jar`, `hadoop-aws-${hadoop-version}.jar`, `aws-sdk-java-${version}` and other necessary dependencies | `gravitino-gcp-{gravitino-version}.jar`, `gcs-connector-${hadoop-version}`.jar, other necessary dependencies | `gravitino-aliyun-{gravitino-version}.jar`, hadoop-aliyun-{hadoop-version}.jar, aliyun-sdk-java-{version} and other necessary dependencies | `gravitino-azure-${gravitino-version}.jar`, `hadoop-azure-${hadoop-version}.jar`, and other necessary dependencies |
+
+For `hadoop-aws-${hadoop-version}.jar`, `hadoop-azure-${hadoop-version}.jar` and `hadoop-aliyun-${hadoop-version}.jar` and related dependencies, you can get them from `${HADOOP_HOME}/share/hadoop/tools/lib/` directory.
+For `gcs-connector`, you can download it from the [GCS connector](https://github.com/GoogleCloudDataproc/hadoop-connectors/releases) for hadoop2 or hadoop3. 
+
+If there still have some issues, please report it to the Gravitino community and create an issue. 
```

Comment:
Let's make the users' life easier by providing a link.
Or else they have to check where is the community and/or where is the repo.

```suggestion
If there are still issues, please consider [filing an issue](...) 
```